src/app/backend/client/api/common.go

```diff
@@ -29,3 +33,14 @@ func ToSelfSubjectAccessReview(namespace, name, resource, verb string) *v1.SelfS
 		},
 	}
 }
+
+// GenerateCSRFKey generates random csrf key
+func GenerateCSRFKey() string {
+	bytes := make([]byte, 256)
+	_, err := rand.Read(bytes)
+	if err != nil {
+		panic("Fatal error. Could not generate csrf key")
```

Comment:
It is common to start error messages with a small letter in Go, so perhaps `panic("could not generate csrf key")` would be better?